tutorials/getting-started/getting-started.md

```diff
@@ -0,0 +1,147 @@
+{% hint style="working" %}
+NOTE: Imagemagick is not strictly required, but it enables cropping and scaling images as well as providing performance improvements for image operations in general, so it is recommended for most installations. \
+{% endhint %}
+
+{% hint style="working" %}
+
+### Troubleshooting
+
+Depending on your exact configuration, you run into some issue related to file locations and paths.
+
+#### I get a "not found" error when I try to run "npm"
+
+Check if NPM was installed with Node by running "npm" from the command line. If you get a "not found" error, install it with this command:
+
+```bash
+    curl -L https://npmjs.org/install.sh | sh
+```
+
+#### I get a "not found" error when I try to run "node"
+
+Node may not have been added to the $PATH when it was installed, if you can't run the "node" command you need to add node's directory to our $PATH in order for it to work its magic.
+
+1. Create a plaintext file called .profile in your home directory if you don't already have one.
+
+2. Add these two lines to .profile: to add Node to the $PATH
+
+    ```markup
+        export NODE_PATH="/usr/local/lib/node"
+        export PATH="/usr/local/share/npm/bin:$PATH"
+    ```
+
+{% endhint %}
+
+
+## Windows Dependency Installation
+
+While Linux and macOS can use a package manager to conveniently install packages for a single location, Windows users will need to manually download and install everything.
+
+#### Install MongoDB
+
+1. Go to [https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/#install-mongodb-community-edition](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/#install-mongodb-community-edition) to download the installer.
+
+2. Run the installer.
+
+3. During the installation process, select a “Complete” installation, and choose to “Install MongoDB as a Service”
+
+#### Install Node
+
+1. Go to [https://nodejs.org/en/download/](https://nodejs.org/en/download/) to download the installer.
+
+2. Run the installer, and Node will be installed.
+
+#### Install Imagemagick
+
+Imagemagick can help speed up image processing for Apostrophe, and adds additional cropping and scaling options for uploaded images. Download and install it [Imagemagick's site](http://www.imagemagick.org/script/download.php#windows).
+
+## Apostrophe Installation (All OSes)
+
+After you have all the dependencies set up, you can get down to setting up Apostrophe.
+
+1. Install the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md):
+
+    ```bash
+        npm install apostrophe-cli -g
+    ```
+
+2. Next, make sure you’re in the directory where you want your Apostrophe projects located and create your project:
+
+    ```bash
+        apostrophe create-project <your-project-name> && cd <your-project-name> && npm i
+    ```
+
+Running `npm-install` will install Apostrophe with the remaining dependencies. Before you can run Apostrophe you’ll want to create an administrative user.
```

Comment:
This is redundant to the `npm i` on the previous line and the hyphen is incorrect.